### PR TITLE
Debug log

### PR DESCRIPTION
**Pull request by RomanIovlev**

### CODE REVIEW

File: jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.epam.jdi.light.driver;
 
 import com.epam.jdi.light.driver.get.DriverTypes;
+import com.epam.jdi.light.settings.WebSettings;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java:4]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.settings.WebSettings'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 16
- Added 14
           

Complexity increasing per file
==============================
- jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIMSelectAssert.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876&bid=18234244)

File: jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java [hunk 2]
```diff
@@ -1,9 +1,12 @@
 package com.epam.jdi.light.actions;
 
+import com.epam.jdi.light.common.Exceptions;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java:3]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.common.Exceptions'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIMSelectAssert.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package com.epam.jdi.light.asserts.generic;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.complex.IMultiSelector;
+import org.hamcrest.Matcher;
+
+import java.util.List;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIMSelectAssert.java:9]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.asserts.core.SoftAssert'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIMSelectAssert.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import static org.hamcrest.Matchers.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIMSelectAssert.java:10]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java [hunk 5]
```diff
@@ -14,8 +14,8 @@
 import static com.epam.jdi.light.elements.init.UIFactory.*;
 import static com.epam.jdi.light.elements.init.entities.collection.EntitiesCollection.*;
 import static com.epam.jdi.light.settings.WebSettings.*;
+import static org.hamcrest.MatcherAssert.*;
```
Comment (codacy-review) [inline jdi-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java:17]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.MatcherAssert'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light-html/src/main/java/com/epam/jdi/light/ui/html/asserts/TextAreaAssert.java [hunk 6]
```diff
@@ -7,7 +7,10 @@
 import org.hamcrest.Matcher;
 import org.hamcrest.Matchers;
 
+import java.util.List;
+
 import static com.epam.jdi.light.asserts.core.SoftAssert.*;
+import static org.hamcrest.Matchers.*;
```
Comment (codacy-review) [inline jdi-light-html/src/main/java/com/epam/jdi/light/ui/html/asserts/TextAreaAssert.java:13]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java [hunk 7]
```diff
@@ -1,9 +1,12 @@
...
 import org.aspectj.lang.ProceedingJoinPoint;
 import org.aspectj.lang.annotation.*;
 
 import static com.epam.jdi.light.actions.ActionHelper.*;
+import static com.epam.jdi.light.common.Exceptions.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java:8]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.common.Exceptions'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java [hunk 8]
```diff
@@ -13,9 +13,10 @@
 import static com.epam.jdi.light.elements.complex.table.TableMatcher.hasValue;
 import static com.epam.jdi.light.elements.complex.table.TableMatcher.*;
 import static com.epam.jdi.light.elements.init.entities.collection.EntitiesCollection.*;
+import static com.epam.jdi.tools.LinqUtils.*;
+import static com.epam.jdi.tools.PrintUtils.*;
```
Comment (codacy-review) [inline jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java:17]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.PrintUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java [hunk 9]
```diff
@@ -13,9 +13,10 @@
 import static com.epam.jdi.light.elements.complex.table.TableMatcher.hasValue;
 import static com.epam.jdi.light.elements.complex.table.TableMatcher.*;
 import static com.epam.jdi.light.elements.init.entities.collection.EntitiesCollection.*;
+import static com.epam.jdi.tools.LinqUtils.*;
```
Comment (codacy-review) [inline jdi-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java:16]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.LinqUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java [hunk 10]
```diff
@@ -15,6 +16,7 @@
 import static com.epam.jdi.light.driver.get.DriverData.*;
 import static com.epam.jdi.light.driver.get.DriverTypes.*;
 import static com.epam.jdi.light.settings.JDISettings.*;
+import static com.epam.jdi.light.settings.WebSettings.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java:19]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.settings.WebSettings'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-bdd-tests/src/test/java/cucmberTests/stepdefs/UserStepdefs.java [hunk 11]
```diff
@@ -16,27 +13,10 @@
 import static io.github.com.entities.Users.*;
 import static io.github.com.pages.Header.*;
 import static java.util.Arrays.*;
-import static org.testng.Assert.*;
+import static org.junit.Assert.*;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucmberTests/stepdefs/UserStepdefs.java:16]: Codacy Issue found: [Avoid unused imports such as 'org.junit.Assert'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java [hunk 12]
```diff
@@ -15,6 +15,7 @@
 import static com.epam.jdi.light.asserts.core.SoftAssert.*;
 import static com.epam.jdi.light.settings.JDISettings.*;
 import static com.epam.jdi.tools.EnumUtils.*;
+import static com.epam.jdi.tools.LinqUtils.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java:18]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.tools.LinqUtils'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/complex/ChecklistAssert.java [hunk 13]
```diff
@@ -8,6 +8,7 @@
 import java.util.List;
 
 import static com.epam.jdi.light.asserts.core.SoftAssert.*;
+import static org.hamcrest.Matchers.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/complex/ChecklistAssert.java:11]: Codacy Issue found: [Avoid unused imports such as 'org.hamcrest.Matchers'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

File: jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java [hunk 14]
```diff
@@ -1,9 +1,12 @@
...
+import static com.epam.jdi.light.settings.WebSettings.*;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java:9]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.settings.WebSettings'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5559876)

**Comment by mf-code** [comment]
[5362] - SNIFyHce

**Comment by mf-code** [comment]
[5362] - sRWAiJWR

**Comment by mf-code** [comment]
[5362] - Allure report on Netlify: https://5ec577b457f9a07cb390178a--musing-swartz-6fc060.netlify.app

**Pull request merged**